en/docs/emr-connectivity/guides/emr-connector.md

```diff
@@ -0,0 +1,84 @@
+## Overview
+The FHIR Client Connector is a versatile tool designed to streamline integration with healthcare systems that adhere to the FHIR standard. Whether you're using Ballerina or a Micro Integrator, this connector provides a unified interface to interact with FHIR-compliant APIs, making it easier to build and manage healthcare applications.
+
+## Connecting with Epic EMR 
+In this example we will look into the steps how can we use the Client Connector to connect with the Epic EMR system. 
+
+### Prerequisites
+1. First we need to create Backend OAuth 2.0 App. To use the client_credentials OAuth 2.0 grant type to authorize the backend application's access to patient information we need 'Client ID' and the 'Public key'. 
+
+    Let's create a Public Private Key Pair using OpenSSL. 
+
+    Creating the private key
+    ```
+    openssl genrsa -out <path_to_key>/privatekey.pem 2048
+    ```
+
+    Creating the public key
+    ```
+    openssl req -new -x509 -key <path_to_key>/privatekey.pem -out <path_to_key>/publickey509.pem -subj '/CN=myapp'
+    ```
+
+2. Login to [EPIC FHIR](https://fhir.epic.com/) with your credentials. 
+3. Navigate to 'Build Apps' tab to create an application. 
```

Comment:
Let's link the epic documentation on creating apps

Reply:
Fixed.